experimental/IntersectionTheory/src/schubert.jl

```diff
@@ -122,11 +111,27 @@ julia> basis(G)
  [c[2], c[1]^2]
  [c[1]*c[2]]
  [c[2]^2]
- 
+
+julia> schubert_classes(G, 2)
+2-element Vector{MPolyQuoRingElem{MPolyDecRingElem{QQFieldElem, QQMPolyRingElem}}}:
+ c[1]^2 - c[2]
+ c[2]
+
 ```
+
 """
 function schubert_classes(G::AbstractVariety)
    get_attribute(G, :grassmannian) === nothing && error("the abstract_variety is not a Grassmannian")
    S, Q = G.bundles
    return [schubert_classes(G, i) for i = 0:rank(S)*rank(Q)]
 end
+
+function schubert_classes(G::AbstractVariety, m::Int)
+  get_attribute(G, :grassmannian) === nothing && error("the abstract_variety is not a Grassmannian")
```

Comment:
So this is just code being moved, so don't take this comment as a blocker for this PR, just as a suggestion for a potential improvement now or in the future: It seems better to use `has_attribute` for testing for the presence of an attribute rather than relying on an implementation detail.
```suggestion
  has_attribute(G, :grassmannian) || error("the abstract_variety is not a Grassmannian")
```

This could also use the `@req` macro, and perhaps have a slightly nicer error:
```suggestion
  @reg has_attribute(G, :grassmannian) "the given abstract variety must be a Grassmannian"
```